source/documentation/tools/create-a-derived-table/project-structure.md

```diff
@@ -1,30 +1,384 @@
+      │          └── stg_xhibit__court.sql
+      ├── courts
+      │   ├── court_intermediate
+      │   │
+      │   ├── court_dimensional_layer
+      │
+      ├── prison  # domain
+      │   ├── prison_intermediate # database
+      │   │      ├──intermediate_models # model
+      │   │      ...
+      │   ├── prison_dimensional_layer
+      │   │
+      │   ├── prison_mart
+```
+
+Data modelling is hard, so if the considerations about domains, databases, or data model structures aren't clear - if you're unsure, reach out to the [data modelling team](https://asdslack.slack.com/archives/C03J21VFHQ9) and we'll do our best to help.
+
+Please note that the use of create-a-derived-table has evolved over time, and best practices change, the reality of our project may be out of sync with the best practices stated here. 
+
+## 2-staging
+
+The staging layer is where our journey begins. This is the foundation of our project, where we bring all the individual components we're going to use to build our more complex and useful models into the project.
+
+We'll use an analogy for working with dbt throughout this guide: thinking modularly in terms of atoms, molecules, and more complex outputs like proteins or cells (we apologize in advance to any chemists or biologists for our inevitable overstretching of this metaphor). Within that framework, if our source system data is a soup of raw energy and quarks, then you can think of the staging layer as condensing and refining this material into the individual atoms we’ll later build more intricate and useful structures with.
+
+### Staging: Files and folders
+
+Let's zoom into the staging directory from our `models` file tree [in the overview](/best-practices/how-we-structure/1-guide-overview) and walk through what's going on here.
+
+```shell
+models/staging
+├── stg_nomis
+│   ├── stg_nomis__docs.md
+│   ├── stg_nomis__models.yml
+│   ├── stg_nomis__offender.sql
+│   └── stg_nomis__prison.sql
+└── stg_xhibit
+    ├── stg_xhibit__models.yml
+    └── stg_xhibit__court.sql
+```
+
+- **Folders.** Folder structure is extremely important in dbt. Not only do we need a consistent structure to find our way around the codebase, as with any software project, but our folder structure is also one of the key interfaces for understanding the knowledge graph encoded in our project (alongside the DAG and the data output into our warehouse). It should reflect how the data flows, step-by-step, from a wide variety of source-conformed models into fewer, richer business-conformed models. Moreover, we can use our folder structure as a means of selection in dbt [selector syntax](https://docs.getdbt.com/reference/node-selection/syntax). For example, with the above structure, if we got fresh xhibit data loaded and wanted to run all the models that build on our xhibit data, we can easily run `dbt build --select staging/xhibit_stg+` and we’re all set for building more up-to-date reports on payments.
```

Comment:
I wrote a comment below about explicating acronyms - looks like this is the first mention of DAG instead of where I wrote my comment